lib/commands/external.js

```diff
@@ -20,47 +22,46 @@ class ExternalCommand {
         });
     }
 
-    static validateEnv(extName) {
+    static checkIfPackageExists(extName) {
         try {
             const path = require('path');
-            const pathForCmd = path.resolve(process.cwd(), 'node_modules', '@webpack-cli', extName);
+            const pathForCmd = path.resolve(process.cwd(), 'node_modules', packagePrefix, extName);
             require.resolve(pathForCmd);
             return pathForCmd;
         } catch (err) {
             return false;
         }
     }
-    // TODO: to implement once "inquirer" will be avialable inside the CI
-    // static async promptInstallation(scopeName, name) {
-    //     const path = require('path');
-    //     const fs = require('fs');
-    //     const isYarn = fs.existsSync(path.resolve(process.cwd(), 'yarn.lock'));
-    //     const packageManager = isYarn ? 'yarn' : 'npm';
-    //     const options = [isYarn ? 'add' : 'install', '-D', scopeName];
-    //
-    //     const commandToBeRun = `${packageManager} ${options.join(' ')}`;
-    //     logger.error(`The command moved into a separate package: ${chalk.keyword('orange')(name)}\n`);
-    //     const question = `Would you like to install ${name}? (That will run ${chalk.green(commandToBeRun)})`;
-    //     const { installConfirm } = await prompt([
-    //         {
-    //             type: 'confirm',
-    //             name: 'installConfirm',
-    //             message: question,
-    //             default: 'Y',
-    //             choices: ['Yes', 'No', 'Y', 'N', 'y', 'n'],
-    //         },
-    //     ]);
-    //     if (installConfirm === true) {
-    //         await ExternalCommand.runCommand(commandToBeRun);
-    //         return ExternalCommand.validateEnv(name);
-    //     }
-    //     // eslint-disable-next-line require-atomic-updates
-    //     process.exitCode = -1;
-    // }
+    static async promptInstallation(packageName, name) {
+        const path = require('path');
+        const fs = require('fs');
+        const isYarn = fs.existsSync(path.resolve(process.cwd(), 'yarn.lock'));
+        const packageManager = isYarn ? 'yarn' : 'npm';
+        const options = [isYarn ? 'add' : 'install', '-D', packageName];
+
+        const commandToBeRun = `${packageManager} ${options.join(' ')}`;
+        logger.error(`The command moved into a separate package: ${chalk.keyword('orange')(packageName)}\n`);
+        const question = `Would you like to install ${name}? (That will run ${chalk.green(commandToBeRun)})`;
+        const { installConfirm } = await prompt([
+            {
+                type: 'confirm',
+                name: 'installConfirm',
+                message: question,
+                default: 'Y',
+                choices: ['Yes', 'No', 'Y', 'N', 'y', 'n'],
+            },
+        ]);
+        if (installConfirm === true) {
```

Comment:
```suggestion
        if (installConfirm) {
```

Reply:
This one seems to be a redundant check :thinking:

Reply:
My question: is `installConfirm` always a boolean?

Reply:
Yes it is.